src/values/float_value.rs

```diff
@@ -64,23 +64,38 @@ impl<'ctx> FloatValue<'ctx> {
         unsafe { FloatValue::new(LLVMConstFNeg(self.as_value_ref())) }
     }
 
+    #[llvm_versions(4.0..15.0)]
     pub fn const_add(self, rhs: FloatValue<'ctx>) -> Self {
+        use llvm_sys::core::LLVMConstFAdd;
+
         unsafe { FloatValue::new(LLVMConstFAdd(self.as_value_ref(), rhs.as_value_ref())) }
     }
 
+    #[llvm_versions(4.0..15.0)]
     pub fn const_sub(self, rhs: FloatValue<'ctx>) -> Self {
+        use llvm_sys::core::LLVMConstFSub;
+
         unsafe { FloatValue::new(LLVMConstFSub(self.as_value_ref(), rhs.as_value_ref())) }
     }
 
+    #[llvm_versions(4.0..15.0)]
     pub fn const_mul(self, rhs: FloatValue<'ctx>) -> Self {
+        use llvm_sys::core::LLVMConstFMul;
+
         unsafe { FloatValue::new(LLVMConstFMul(self.as_value_ref(), rhs.as_value_ref())) }
     }
 
+    #[llvm_versions(4.0..15.0)]
     pub fn const_div(self, rhs: FloatValue<'ctx>) -> Self {
+        use llvm_sys::core::LLVMConstFDiv;
+
         unsafe { FloatValue::new(LLVMConstFDiv(self.as_value_ref(), rhs.as_value_ref())) }
     }
 
+    #[llvm_versions(4.0..15.0)]
```

Comment:
These cfgs are redundant. Are const methods getting removed in 16?

Reply:
Removed in 15. Please check the changelog.

Reply:
I can change these to inclusive ranges if you prefer. I prefer them too, but was following the style of the rest of the crate.

Reply:
I don't have a preference. Feel free to use inclusive if you wish

Reply:
Do you have any background or links about why they were removed? I didn't find much

Reply:
Ah okay the changelog mentions the non const versions being able to const fold

Reply:
Changed to inclusive ranges for clarity.